.github/workflows/docker-build-push.yml

```diff
@@ -0,0 +1,68 @@
+# 工作流名称
+name: Build and Push Docker Image
+
+on:
+  push:
+    tags: "*"
+    branches: ["main"]
```

Comment:
why do we need push action on main branch? Would pull request be better?

Reply:
The triger strategy has been modified, and the image build-push pipeline will only be triggered when tag(release) occurs.